lib/bin/lwm2m_carrier/include/lwm2m_os.h

```diff
@@ -241,14 +243,17 @@ int lwm2m_os_pdn_deactivate(uint8_t cid);
  *
  * @param cid The context ID of the PDN connection.
  *
- * @retval  0      If success.
+ * @return A non-negative PDN ID on success, or a negative errno otherwise.
```

Comment:
```suggestion
 * @retval  A      non-negative PDN ID on success, or a negative errno otherwise.
```
To render like values instead of a text?